util/Pos/Util/LoggerConfig.hs

```diff
@@ -141,7 +141,18 @@ instance FromJSON LoggerTree where
         (_ltMinSeverity :: Severity) <- o .: "severity" .!= Debug
         return LoggerTree{..}
 
-instance Semigroup LoggerTree
+mkUniq :: [LogHandler] -> [LogHandler]
```

Comment:
@erikd LoggerConfig can be composed; this is to make sure no duplicated backends will be setup.